pkg/cache/placeholder_manager.go

```diff
@@ -97,3 +98,22 @@ func (mgr *PlaceholderManager) setMockedClients(mockedClients *client.Clients) {
 	defer mgr.Unlock()
 	mgr.clients = mockedClients
 }
+
+func (mgr *PlaceholderManager) cleanOrphanPlaceholders() {
+	for taskID, pod := range mgr.orphanPod {
+		err := mgr.clients.KubeClient.Delete(pod)
+		if err == nil {
+			delete(mgr.orphanPod, taskID)
+		}
+	}
+}
+
+func (mgr *PlaceholderManager) Start() {
```

Comment:
we need a stop function for this too.
to stop this, you probably need a stop chan to help

Reply:
ok,done.